CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
+
+- Initial version.
```

Comment:
We can copy this from the first commit in the relic repo. I think the version here should be 0.1.0 right?

Reply:
Yes!